src/System.IO.Compression.ZipFile/ref/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      netfx;
```

Comment:
Just double check that this doesn't change what is included for net461.

Reply:
It turns out we were overriding the net461 with this change, so with my latest change that is now fixed.